### DISCUSSION TITLE
Run bunx prettier with latest prettier version

### DISCUSSION THREAD

**Discussion by haunt98** [oven-sh/bun#3627] [Q&A] [answered]
prettier recently release version `3.0.0` but bunx prettier in my local still show `2.8.8`.

I usually run like this:

```sh
bunx prettier --write .
```

**Comment by charnould** [comment]
Did you try:

`bunx prettier@latest --write .` (it works for `npx`)

**Reply by haunt98** [reply]
Thanks @charnould, as @Jarred-Sumner answer below, bunx cache version in `/tmp` but I never restart so.

**Comment by Jarred-Sumner** [comment, marked as answer]
bunx updates whenever `/tmp` is cleared, so if you restart your computer it should update. But if you never restart it will basically never update. You can force a version via `@` (as @charnould said)

**Reply by haunt98** [reply]
Thanks @Jarred-Sumner, looks like this is the root cause because I never restart my laptop :D
